tools/scrape/github.py

```diff
@@ -216,7 +216,7 @@ def scrape_plugin_repos(num):
     query = r.table('plugin_github_repos').filter({'is_blacklisted': False})
 
     # We don't want to scrape forks that not many people use.
-    query = query.filter(r.not_((r.row['is_fork'] == True) & (
+    query = query.filter(r.not_((r.row['is_fork'] is True) & (
```

Comment:
I think this may break the query. Even if it doesn't, this change reads as less idiomatic use of the RethinkDB driver. `flake8`'s error doesn't really apply here, as using `==` is stylistically correct in this case.

Many database libraries like this one use Python's `__eq__` and other magic methods to allow for elegant query building. This is done by translating the [method call](https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/ast.py#L138-L139) into a [query operator](https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/ast.py#L888-L890).